Added 'New year's Campaign 2020' in-app message

IndrekV opened the pull request:
### 📒 Description
Add in-app message for new year's campaign

### 🤯 List of changes
Adds text to for new year's campaign

### 👫 Relationships

Closes #3612 

### 🔎 Review hints

- Delete the value in database table `settings.message_seen`
- Change the JSON file url in `context.cc` line `1556` to `https://raw.githubusercontent.com/toggl-open-source/toggldesktop/add-iam/new-year-campaign-2020/releases/message.json`
- Start app and see if message pops up and if link works


**Not to be merged before stable release of the feature!**

skel35 commented:
To test this, In addition to the steps above, you should also change the time on your computer to after 12/17/2019 @ 9:00am (UTC)

NghiaTranUIT commented:
It's a only one remaining task for In-app message. Could we proceed?

IndrekV commented:
I added the type param to make it work with the current solution. Also, I updated the logic for future release so that we don't need to define a `type` when there is no `appversion` param present (5c595dd42456a28aa9510b78e22c5e81388787b9)

Let's merge this on Monday when we are sure the web page the action button is pointing is live.
